yeat/cli.py

```diff
@@ -79,6 +81,16 @@ def run(
         raise RuntimeError("Snakemake Failed")
 
 
+def check_positive(value):
+    try:
+        value = int(value)
+        if value <= 0:
+            raise argparse.ArgumentTypeError(f"{value} is not a positive integer")
+    except ValueError:
+        raise argparse.ArgumentTypeError(f"{value} is not an integer")
```

Comment:
>Using ValueError here would be more appropriate than Exception.

I tried to raise a `ValueError`; however, every time the exception is thrown, the `argparse.ArgumentTypeError` is thrown instead. It seems like it is because the function, `check_positive` is called in the CLI. I tried a variety of ways to catch the ValueError for testing, but I was unable to figure out how to do so.